tests/Fixtures/Kernel.php

```diff
@@ -58,6 +59,10 @@ protected function configureContainer(ContainerBuilder $c, LoaderInterface $load
             ->setAutoconfigured(true)
             ->setAutowired(true)
         ;
+        $c->register(CommentEventSubscriber::class)
```

Comment:
You'll have tag this as an event subscriber manually instead of relying on autoconfiguration. Early versions of doctrine bundle don't register this interface for autoconfiguration.